PWGHF/HFC/Tasks/taskFlow.cxx

```diff
@@ -1052,6 +1132,207 @@ struct HfTaskFlow {
           //}
         }
 
+        if (sameEvent && (loopCounter == 1)) {
+          // FILL USUAL MFT DISTRIBUTIONS
+          registry.fill(HIST("Data/TpcMft/kCFStepAll/hEta"), eta2);
+          registry.fill(HIST("Data/TpcMft/kCFStepAll/hPhi"), phi2);
+        }
+
+      } // end of loop over tracks2
+    } // end of loop over tracks 1
+  }
+
+  template <CorrelationContainer::CFStep step, typename TTarget, typename TTracksTrig, typename TTracksAssoc>
+  void fillCorrelationsReassociatedMftTracks(TTarget target,
+                                             TTracksTrig const& tracks1, TTracksAssoc const& tracks2,
+                                             float multiplicity, float posZ, bool sameEvent, bool cutAmbiguousTracks)
```

Comment:
Why is `step` a template argument?

Reply:
That means that somewhere in the code, you give an `int` value instead of a `CorrelationContainer` value. I cannot help unless I see the code.